src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/GovUk.Education.ExploreEducationStatistics.Public.Data.Api.csproj

```diff
@@ -53,5 +53,26 @@
     <ProjectReference Include="..\GovUk.Education.ExploreEducationStatistics.Public.Data.Services\GovUk.Education.ExploreEducationStatistics.Public.Data.Services.csproj"/>
     <ProjectReference Include="..\GovUk.Education.ExploreEducationStatistics.Public.Data.Utils\GovUk.Education.ExploreEducationStatistics.Public.Data.Utils.csproj"/>
   </ItemGroup>
+  
+  <Target Name="BuildOpenApiFiles" AfterTargets="Build">
+    <!-- Add new API versions to this group -->
```

Comment:
I don't quite understand what the purpose of this block is?

Reply:
In the future, when we want to publish v1+ API versions, we would add extra lines for each version e.g.

```xml
<ItemGroup>
  <Version Include="1" />
  <Version Include="2" />
</ItemGroup>
```

Unfortunately, the Swashbuckle CLI doesn't have an option to generate all the available OpenAPI docs automatically. You have to manually provide the name of each document (i.e. the API version number) to the CLI.

After thinking about it some more, I went down a rabbit hole trying to create a custom MSBuild task that could fetch the API version numbers for us and generate the docs. 

Unfortunately, after trying a few things, I haven't been able to get the custom task working. I think in the interest of time, whilst not ideal, we stick with the current solution of adding these version numbers manually.